src/sentry/services/hybrid_cloud/__init__.py

```diff
@@ -38,6 +38,9 @@ class ApiProjectKey:
 class ApiOrganization:
     slug: str = ""
     id: int = -1
+    # True iff the organization was queried with a user_id context, and that user_id
```

Comment:
Not, 100% sure yet about this.  I was hoping to include some amount of organization member context in the default organization object so that many of the organization calls have enough context for a given user id their relationship to it.  This `is_member` may expand to `relation_to_user: ApiOrganizationMember`

Reply:
I think having a sketch of the membership is likely more useful than a boolean. If possible emulating the existing Organization record interface might help us avoid needing to churn more code.